.github/diff_report/diff_screenshots.py

```diff
@@ -0,0 +1,137 @@
+parser.add_argument("output_dir", type=str, help="Directory to save the screenshots diff report")
+
+args = parser.parse_args()
+
+# "before" and "after" directories are named: artifacts/$TARGET_BRANCH and artifacts/$BRANCH_NAME
+before_branch_name = args.before_dir.split(os.path.sep)[-1]
+after_branch_name = args.after_dir.split(os.path.sep)[-1]
+
+def list_files_recursively(path: str) -> list[str]:
+    """ Return a list of paths to all png files in the directory tree """
+    return glob.glob(path + "/**/*.png", recursive=True)
+
+
+def compute_file_hash(file_path: str) -> str:
+    """ Return the file hash using sha256 """
+    hash_func = hashlib.new('sha256')
+    
+    with open(file_path, 'rb') as file:
+        while chunk := file.read(8192):  # Read the file in chunks of 8192 bytes
+            hash_func.update(chunk)
+    
+    return hash_func.hexdigest()
+
+
+def get_pathname_fragment(path:str) -> str:
+    """ Extract the last 3 parts of the path:
+            en/tools_views/ToolsCalcFinalWordDoneView.png
+
+        These paths will be the same in the "before" and "after" directories.
+    """
+    parts = path.split(os.path.sep)
+    if len(parts) < 3:
+        raise ValueError(f"Path should have at least 3 parts: {path}")
+    return os.path.sep.join(parts[-3:])
+
+
+def get_locale_and_screenshot_name(path: str) -> tuple[str, str]:
+    """ Parse the path to extract the locale and the screenshot name.
+
+        Assumes we're working with a path like:
+            en/tools_views/ToolsCalcFinalWordDoneView.png
+    """
+    parts = path.split(os.path.sep)
+    if len(parts) != 3:
+        raise ValueError(f"Path should have 3 parts: {path}")
+    return parts[0], parts[-1].split(".")[0]
+
+
+# Recursively list and hash all png files in the "before" directory
+before_screenshots = {}
+paths_before = []
+for file in list_files_recursively(args.before_dir):
+    screenshot_path = get_pathname_fragment(file)
+    before_screenshots[screenshot_path] = compute_file_hash(file)
+    paths_before.append(screenshot_path)
+
+# Do the same for the "after" directory, but do the diff while we're here
+only_in_after = []
+diffs: list[str] = []
+paths_after = []
+for file in list_files_recursively(args.after_dir):
+    screenshot_path = get_pathname_fragment(file)
+    if screenshot_path not in before_screenshots:
+        only_in_after.append(screenshot_path)
+
+    elif before_screenshots[screenshot_path] != compute_file_hash(file):
+        diffs.append(screenshot_path)
+    
+    paths_after.append(screenshot_path)
+
+only_in_before = set(paths_before) - set(paths_after)
+
+html_content = "<h1>Screenshots diff report</h1>"
+html_content += f"""<p>Comparing {before_branch_name} to <a href="https://github.com/SeedSigner/seedsigner-translations/compare/dev...kdmukai:seedsigner-translations:{after_branch_name}" target="github">{after_branch_name}</a></p>"""
```

Comment:
I'm pretty sure `kdmukai` should not be hard coded here. I believe an env variable needs to be added to the workflow and passed into the python script.

```
env:
  REPO_NAME: ${{ github.event.repository.name }}
```